packages/cyberstorm/src/components/PackageCard.tsx

```diff
@@ -132,6 +134,7 @@ function getPackageFlags(
   if (isPinned) {
     flagList.push(
       <PackageFlag
+        key={1}
```

Comment:
React recommends using "meaningful" strings as keys, so here we could use the label texts. Although it might mostly be based on using indexes as keys, where change in order of things might degrade performance, and thus might not apply to manually set numbers. (But then again, if order of these things are changed manually, you'd have to change the keys too or they would be illogical.)

Reply:
switched to "flag_pinned" etc. also adde "category_" prefix for category tag's key

Reply:
I did this fix